assets/stylesheets/style.css

```diff
@@ -0,0 +1,652 @@
+@font-face {
+  font-family: "headerFont";
+  src: url("../fonts/Cocogoose/Cocogoose Pro Italic-trial.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "textFont";
+  src: url("../fonts/Lato2OFL/Lato-Semibold.ttf");
+}
+
+@font-face {
+  font-family: "capFont";
+  src: url("../fonts/Lato2OFL/Lato-Bold.ttf");
+}
+
+@font-face {
+  font-family: "inHead";
+  src: url("../fonts/Lato2OFL/Lato-Black.ttf");
+}
+
+:root {
+  --image: url("../images/top_back.png");
+}
+
+.light {
+  background: white;
+  color: #272a31;
+}
+
+.light .my-img {
+  content: url("../images/top_logo.png");
+}
+
+@media (max-width: 768px) {
+  .light {
+    background: #272a31;
+    color: white;
+  }
+
+  .light .my-img {
+    content: url("../images/inside_logo.png");
+  }
+}
+
+.dark {
+  background: #272a31;
+  color: white;
+}
+
+.dark .my-img {
+  content: url("../images/inside_logo.png");
+}
+
+@media (max-width: 768px) {
+  .dark {
+    background: white;
+    color: #272a31;
+  }
+
+  .dark .my-img {
+    content: url("../images/top_logo.png");
+  }
+}
+
+body {
+  box-sizing: border-box;
+  overflow-x: hidden;
+  background: linear-gradient(to bottom, black, white, black);
+}
+
+body h2 {
+  margin-bottom: 2px;
+}
+
+body ul {
+  list-style: none;
+}
+
+body hr.red {
+  max-width: 8%;
+  margin: 0 auto;
+  margin-bottom: 2%;
+  border-top: 2px solid red;
+}
+
+.content > * {
+  margin-inline-start: 10%;
+}
+
+body .right > * {
+  margin: 2% 0;
+}
+
+body hr.inside {
+  max-width: 40%;
+  border-top: 2px solid;
+}
+
+body #imp {
+  text-align: center;
+  border: 3px solid #ec5242;
+  border-radius: 5px;
+  padding: 0;
+}
+
+.top {
+  display: none;
+}
+
+nav {
+  background: linear-gradient(to right, lightyellow, white);
+}
```

Comment:
As requested in the project requirements:
![image](https://user-images.githubusercontent.com/42799579/210622178-11fd3e5e-92f1-474c-8a42-f571aafb2b75.png)

Please make sure to follow the original design colors. Keep in mind that your webpage doesn't need to be identical, but remember that you are following design guidelines! This project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉. 

For this reason, kindly remove the gradients from the headers and update it background colors to look accordingly to the original idea:

Your design:
![image](https://user-images.githubusercontent.com/42799579/210622442-e57a4548-93b5-4daa-9865-531117d42df7.png)

- Top header: black gradient background and black text
- Bottom header: yellow gradient background

Original design:
![image](https://user-images.githubusercontent.com/42799579/210622563-7e6d2084-3a97-46cb-8b48-7b8d68473384.png)

- [ ] Top header: solid black background and white text
- [ ] Bottom header: solid white background